registrations/management/commands/report_pmtct_risks.py

```diff
@@ -115,7 +122,7 @@ def add_to_result(risk, reg):
                 validated=True)
 
             for registration in registrations.iterator():
-
+                add_personally_identifiable_fields(registration)
```

Comment:
hmmmmm ok. that complicates things a little bit if we decide we do want to save it but it should be fine. Maybe add a comment in the task so that it's clear the changes made there aren't saved?